subscription_mass_mailing/models/subscription.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2019 Coop IT Easy SCRL fs
+#   Robin Keunen <robin@coopiteasy.be>
+#   Houssine Bakkali <houssine@coopiteasy.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, api
+
+
+class Subscription(models.Model):
+    _inherit = "subscription"
+
+    @api.multi
+    def message_get_default_recipients(self):
```

Comment:
I don't see the function  being used in the code.

Reply:
ok I removed it